server/src/models/User.js

```diff
@@ -27,11 +27,13 @@ class User extends uniqueFunc(Model) {
   static get jsonSchema() {
     return {
       type: "object",
-      required: ["email"],
+      required: ["email","username"],
 
       properties: {
         email: { type: "string", pattern: "^\\S+@\\S+\\.\\S+$" },
-        cryptedPassword: { type: "string" },
+        username: { type: "string"},
+        admin: {type: "boolean"},
```

Comment:
braces should be space padded
https://github.com/airbnb/javascript#whitespace--in-braces
```js
{ type: "boolean" }
```